packages/fxa-react/components/Portal/index.tsx

```diff
@@ -39,9 +39,12 @@ const Portal = ({
     el = document.createElement('div');
     el.setAttribute('class', 'portal');
     el.setAttribute('id', id);
-    el.setAttribute('role', 'dialog');
     document.body.appendChild(el);
 
+    if (id !== 'alert-bar-portal') {
+      el.setAttribute('role', 'dialog');
+    }
```

Comment:
I know this file is getting a little gross. When payments uses Tailwind we can better combine `DialogMessage` and `Modal` to make this a bit cleaner...

Reply:
Possibly for a future iteration, but could we just use add a `role` prop to the component and set it directly instead of inferring it?

Reply:
Probably, but yeah it makes sense to address in another PR since this is used in payments, I can add that to the ticket I'll make for cleaning this file up.